core/src/main/java/io/lionweb/lioncore/java/serialization/NodePopulator.java

```diff
@@ -56,7 +56,9 @@ class NodePopulator {
         .getElements()
         .forEach(
             element -> {
-              autoResolveMap.put(element.getName(), element);
+              // See
+              // https://lionweb.io/specification/2024.1/metametamodel/metametamodel.html#predefined-builtins-keys
+              autoResolveMap.put("LionWeb.LionCore_builtins." + element.getName(), element);
```

Comment:
Maybe do the same for M3 elements?
https://lionweb.io/specification/2024.1/metametamodel/metametamodel.html#predefined-m3-keys